Hakawai/Core/HKWTextView.m

```diff
@@ -26,11 +26,12 @@ @interface HKWTextView () <UITextViewDelegate, HKWAbstractionLayerDelegate>
 @property (nonatomic) NSMutableDictionary *simplePluginsDictionary;
 
 @property (nonatomic, readwrite) BOOL wasPaste;
+@property (nonatomic, readwrite) NSInteger generalPasteboardChangeCount;
```

Comment:
let's add a comment on what it is it for as it doesn't seem obvious

Reply:
`assign`?

Reply:
yes. will add comment.

but i believe “assign” is the default value, so doesn’t need to be specified